### PR TITLE
User can reply to message

### PR DESCRIPTION
**Pull request by EevanR**
[PT](https://www.pivotaltracker.com/n/projects/2423721)

Checks functionality of replying to a message, sad path and happy path.

### CODE REVIEW

File: features/user_can_reply_to_message.feature [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        | dummy1 | user1@mail.com | password | 1  |
+        | dummy2 | user2@mail.com | password | 2  |
+
+    And the inbox has content
+    And I am logged in as "dummy2"
+    And I am in the inbox
+
+  Scenario: User replies to message [Happy path]
+    When I click on "View"
+    And I fill in "Reply Message" with "reply"
```
Comment (oliverochman) [inline features/user_can_reply_to_message.feature]: please have something else as the reply

File: features/user_can_reply_to_message.feature [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    And I click on "Reply"
+    Then I should see "Your reply message was successfully sent!"
+
```
Comment (oliverochman) [inline features/user_can_reply_to_message.feature]: please add another assertion here about seeing the reply in the conversation

File: features/user_can_reply_to_message.feature [hunk 3]
```diff
@@ -0,0 +1,26 @@
+Feature: user can reply to message
+    As a user
+    In order to have conversations with my friends
+    I want to be able to send replies to messages
+
+    Background:
+      Given the following user exist:
+
```
Comment (oliverochman) [inline features/user_can_reply_to_message.feature]: please remove the empty line here

File: features/user_can_reply_to_message.feature (hunk 1 above)
Comment (EevanR) [reply features/user_can_reply_to_message.feature]: ok

File: features/user_can_reply_to_message.feature (hunk 3 above)
Comment (EevanR) [reply features/user_can_reply_to_message.feature]: ok

File: features/user_can_reply_to_message.feature (hunk 2 above)
Comment (EevanR) [reply features/user_can_reply_to_message.feature]: done

**Pull request merged**